src/Spryker/Zed/RabbitMq/Communication/Plugin/RabbitMqQueueMessageCheckerPlugin.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
```

Comment:
[This](https://github.com/spryker/rabbit-mq/blob/master/.license) license does not match what we have in https://github.com/spryker/rabbit-mq/blob/master/LICENSE

Please align with the business which license we should have for this module, and either update https://github.com/spryker/rabbit-mq/blob/master/.license (and all file headers - we have a console command for that) or https://github.com/spryker/rabbit-mq/blob/master/LICENSE